DataFormats/HepMCCandidate/BuildFile.xml

```diff
@@ -2,6 +2,10 @@
 <use   name="DataFormats/Common"/>
 <use   name="DataFormats/StdDictionaries"/>
 <use   name="root"/>
+<use   name="gcc-f77compiler"/>
```

Comment:
we should find out why explicit fortran deps is needed here. This will break e.g. ICC builds as it will interfere with ifort compiler.